Statut HSTrójmiasto.tex

```diff
@@ -0,0 +1,254 @@
+	    \item Oba terminy Walnego Zebrania Członków w trybie nadzwyczajnym muszą być odległe od siebie przynajmniej o 3, ale nie bardziej niż o 7 dni kalendarzowych.
+	\end{enumerate}
+	\end{enumerate}
+	\begin{enumerate}
+	    \item Zarząd jest powołany do kierowania całą działalnością Stowarzyszenia zgodnie z uchwałami Walnego Zebrania Członków.
+	    \item W skład Zarządu wchodzi 3 równorzędnych członków.
+	    \item Do kompetencji Zarządu należą:
+	      \begin{enumerate}
+	        \item realizacja celów Stowarzyszenia,
+	        \item wykonywanie uchwał Walnego Zebrania Członków,
+	        \item sporządzanie planów pracy i budżetu,
+	        \item sprawowanie zarządu nad majątkiem Stowarzyszenia,
+	        \item podejmowanie uchwał o zarządzaniu majątkiem Stowarzyszenia,
+	        \item reprezentowanie Stowarzyszenia na zewnątrz,
+	        \item zwoływanie Walnego Zebrania Członków,
+	        \item przyjmowanie i skreślanie członków zwyczajnych,
+	        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
+	        \item sporządzanie rocznego sprawozdania finansowego,
+	        \item zwalnianie w uzasadnionych przypadkach Członków Zwyczajnych z obowiązku płacenia składek w całości bądź w części.
+	      \end{enumerate}
+	    \item Posiedzenia Zarządu odbywają się w miarę potrzeb, nie rzadziej jednak niż raz na rok.
+	    \item Posiedzenia Zarządu zwołuje:
+	      \begin{enumerate}
+	        \item dwóch członków zarządu działających łącznie,
+	        \item Komisja Rewizyjna.
+	      \end{enumerate}
+	    \item Członek Zarządu może zrezygnować z pełnienia funkcji poprzez złożenie oświadczenia na Walnym Zebraniu Członków.
+	\end{enumerate}
+    \begin{enumerate}
+	    \item Komisja Rewizyjna powoływana jest do sprawowania kontroli nad działalnością Stowarzyszenia.
+	    \item Komisja Rewizyjna składa się z 3 równorzędnych członków.
+	    \item Do kompetencji Komisji Rewizyjnej należy:
+	      \begin{enumerate}
+	        \item kontrolowanie działalności Zarządu,
+	        \item składanie sprawozdań z kontroli na Walnym Zebraniu Członków,
+	        \item składanie wniosków o udzielenie absolutorium dla Zarządu Stowarzyszenia,
+	        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
+	        \item zatwierdzanie rocznego sprawozdania finansowego.
+	      \end{enumerate}
+	    \item Członek Komisji Rewizyjnej może zrezygnować z pełnienia funkcji poprzez złożenie oświadczenia na Walnym Zebraniu Członków.
+	\end{enumerate}
+\section{MAJĄTEK I FUNDUSZE}
+  \begin{enumerate}
+    \item Majątek Stowarzyszenia składa się z:
+      \begin{enumerate}
+        \item składek członkowskich,
+        \item subwencji, darowizn, spadków i zapisów,
+        \item wpływów z odpłatnej działalności statutowej,
```

Comment:
Tam dalej jest, że nie prowadzimy działalności gospodarczej, czy możemy np. sprzedawać miejsce w racku as in kolokacja zgodnie ze statutem?

Reply:
Nie mozemy. Dzialalnosc ma HSwaw by robic takie smieszne rzeczy ale to im sie nie oplaca i maja for fun. Jak juz bedziemy stow co zyje i nie umiera to sie mozna bawic w takie rzeczy ale biurokracji masa za to jest.